wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/DeviceDetailsView.swift

```diff
@@ -127,6 +121,18 @@ struct DeviceDetailsView: View {
         .onDisappear {
             dismissedView?()
         }
+        .sheet(isPresented: $isCertificateViewPresented) {
+            if let certificate = viewModel.e2eIdentityCertificate {
+                E2EIdentityCertificateDetailsView(
+                    certificateDetails: certificate.certificateDetails.uppercased(),
+                    isDownloadAndCopyEnabled: viewModel.isCopyEnabled, isMenuPresented: false
+                ) {
+                    viewModel.downloadE2EIdentityCertificate()
```

Comment:
what does this closure do? Should we keep the named param for readability?

Reply:
kept the named parameter